pkg/netpol/connlist/connlist.go

```diff
@@ -400,6 +407,38 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine) ([]Peer2Pe
 			res = append(res, connectionObj)
 		}
 	}
+
+	// analyze ingress connections - create connection objects for relevant ingress analyzer connections
+	if !ia.IsEmpty() {
```

Comment:
avoid having most of the code in the inner indentation, use the negation of the current condition